A document about our Zulip channel (#19)
anitab-org/documentation: README.md

> # open-source-teams :wave:

Reviewer: can you remove this is replace by the line you removed above please?
```suggestion
# AnitaB.org Open Source Documentation
```

Place to document everything about our AnitaB.org Open Source teams

# Hello :eyeglasses: :v: 
# [About AnitaB.org](https://anitab.org/)
At AnitaB.org, we’re committed to increasing the influence of women on all aspects of technology. Our Local community expands our efforts globally to help individuals all over the world — especially those who are considering or currently pursuing technical careers — can access the resources they need to reach their highest potential.

> Members of the global AnitaB.org Local community network organize events and provide one another with resources to navigate careers in tech. They organize valuable meetups, codeathons, and one-day HopperX1 events modeled after the Grace Hopper Celebration.

Reviewer: this can be added as part of another issue to add information introducing this community on README

Documents:

- [Our AnitaB.org Open Source blog](/our-blog.md)

> 

Reviewer: ```suggestion
- [Zulip: our communication channel](/zulip.md)
```

> For Information regarding our Zulip channel, Zulip /README.md

Reviewer: ```suggestion
```
If you add the line in the comment above then this line is not needed :)

# AnitaB.org Open Source Documentation

> Place to document everything about our AnitaB.org Open Source teams

Reviewer: ```suggestion
This is the place to document everything about our AnitaB.org Open Source community.
```
Its not just about the teams anymore ;)
